src/content/deployment/flavors.md

```diff
@@ -1,165 +1,24 @@
 ---
-title: Create flavors of a Flutter app
-short-title: Flavors
+title: Set up Flutter flavors for Android
+short-title: Flavors (Android)
 description: >
   How to create build flavors specific to different
   release types or development environments.
 ---
 
-## What are flavors
+## Overview
 
-Have you ever wondered how to set up different environments in your Flutter app?
-Flavors (known as _build configurations_ in iOS and macOS), allow you (the developer) to
-create separate environments for your app using the same code base.
-For example, you might have one flavor for your full-fledged production app,
-another as a limited "free" app, another for testing experimental features, and so on.
+A Flutter flavor is basically a collection of settings that
+define how a specific version of your app should be built
```

Comment:
FWIW I think you should describe flutter flavors as either the unified term for various platform specific features or say that flutter flavor is a concept that is implemented by various platform specific features. 

I think it will help readers intuit what to expect.

Reply:
I agree, Reid.

Reply:
I've made this change, but I think we need more discussion around this. I'm getting the sense that what a flavor means for an iOS app is different for what a flavor means for an Android app. @reidbaker, I'll sync up with you to get this figured out on the Android side of things soon. I'm also thinking it's good that we split these two platforms up into different docs.